config/petra_parameters.base

```diff
@@ -0,0 +1,15 @@
+[source]
+class = TINEAttributeSource
+device_address = /PETRA/GLOBALS/keyword
+
+[sink]
+class = InfluxDBSink
+database = petra
```

Comment:
Because of the PW, I am not sure if the config files should be checked into this repository. I think it would better to have the configuration in puppet instead.